apps/frontend/src/pages/quiz/libs/components/onboarding/onboarding.tsx

```diff
@@ -132,13 +116,13 @@ const Onboarding: React.FC = () => {
 									<OnboardingAnswer
 										control={control}
 										key={answer.id}
-										name="answer"
+										name={`answer${question.id.toString()}`}
 										options={answerOptions}
 									/>
 								);
 							})}
 							<div className={styles["button-container"]}>
-								{!isFirstQuestion && (
+								{currentQuestionIndex !== ZERO && (
 									<div className={styles["button-wrapper"]}>
 										<Button
 											label={ButtonLabel.BACK}
```

Comment:
Look below. Even though submit button has isDisabled={!isValid} property, It doesn't work now. Button is enabled even when user didn't answer the question.